### DISCUSSION TITLE
(Mass) Deletion of change requests

### DISCUSSION THREAD

**Discussion by Justman100** [GitbookIO/community#647] [Feature Requests]
That one can delete change requests (in bulk)

**Comment by aikoii** [comment]
Hey @Justman100, thanks for the suggestion!  While deleting change requests in bulk might seem handy, it can also lead to some lost content down the line. That's why we take a similar approach to GitHub's PRs - no deleting, but you can always archive them.

**Reply by Justman100** [reply]
> Hey @Justman100, thanks for the suggestion! While deleting change requests in bulk might seem handy, it can also lead to some lost content down the line. That's why we take a similar approach to GitHub's PRs - no deleting, but you can always archive them.

That's why one doesn't give permissions to just any user! If one does not need changes, one should be able to delete them! Even in bulk! In that case, one simply add separate permissions that one must have in order to be able to do this
